src/wallet/rpcwallet.cpp

```diff
@@ -1294,8 +1294,13 @@ static void MaybePushAddress(UniValue & entry, const CTxDestination &dest)
  * @param  ret            The UniValue into which the result is stored.
  * @param  filter_ismine  The "is mine" filter flags.
  * @param  filter_label   Optional label string to filter incoming transactions.
+ * @param  afterVOut      start adding after afterVOut
```

Comment:
`afterVOut` is not part of the declaration?

Reply:
Should get removed. Left out from some dead coding.